internal/terraform/exec/exec_test.go

```diff
@@ -57,21 +62,29 @@ func TestExec_cancel(t *testing.T) {
 	t.Fatalf("expected cancel error: %#v, given: %#v", expectedErr, err)
 }
 
-func newExecutor(t *testing.T) TerraformExecutor {
-	tmpDir := TempDir(t)
-	workDir := filepath.Join(tmpDir, "workdir")
-	err := os.MkdirAll(workDir, 0755)
-	if err != nil {
-		t.Fatal(err)
-	}
-	installDir := filepath.Join(tmpDir, "installdir")
-	err = os.MkdirAll(installDir, 0755)
+func newExecutor(t *testing.T, tfVersion string) TerraformExecutor {
```

Comment:
Is there a particular reason for adding the version as an argument here? IMO we always want to test everything within this (`exec`) package against the same version, so I'd just set the version in one place.

Reply:
Oh, now I understand the reason to have version hardcoded in the `newExecutor` function. I only made it a variable to be aligned with another function that creates executors. I will update this one to have a static version.